src/components/NcListItemIcon/NcListItemIcon.vue

```diff
@@ -125,11 +130,14 @@ export default {
 		},
 
 		/**
-		 * Icon class to be displayed at the end of the component
+		 * Object containing the material design icon `component`
+		 * and the descriptive `title`
+		 *
+		 * @type {{ component: import('vue').Component, title: string }}
```

Comment:
Nope, let's not pass components as references. This create issues.
See https://github.com/nextcloud/activity/pull/981#discussion_r1005817299